src/docs/cookbook/testing/integration/introduction.md

```diff
@@ -250,6 +253,16 @@ void main() {
       // Then, verify the counter text is incremented by 1.
       expect(await driver.getText(counterTextFinder), "1");
     });
+
+    test('increments the counter during animation', () async {
+      await driver.runUnsynchronized(() async {
+        // First, tap the button.
+        await driver.tap(buttonFinder);
+
+        // Then, verify the counter text is incremented by 1.
+        expect(await driver.getText(counterTextFinder), "1");
+      });
+    });
```

Comment:
this test is explained a bit later, I included it in the complete example so it is easier to extract later